docs/contributing/reference/contributors/index.rst

```diff
@@ -2,3 +2,4 @@ List of contributors
 --------------------
 
 - `@drvinceknight <https://github.com/drvinceknight>`_
+- `@11michais11 <https://github.com/11michalis11>`_
```

Comment:
Great, thanks @11michalis11 looks like you have a small typo and `@11michais11` should be: `@11michalis11` (the `l`) is missing.